docs/usermanual/index.md

```diff
@@ -17,16 +17,161 @@ TODO
+aws_access_key_id = zzz
+aws_secret_access_key = zzz
+```
 
 ### Disk snapshot
 
-TODO
+Use case: you have a volume `vol1` in an AWS account (Account A, in zone `us-east-2b`) which you would like to make a copy 
+of in a different account (Account B).
+
+#### Using the library and the CLI
+
+To make the copy, import the `forensics` package and use the `CreateVolumeCopy` method.
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the volume 'vol1' from one account to another 
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  volume_id='vol1', 
+                                  src_profile='src_profile', 
+                                  dst_profile='analysis_profile')
+```
+
+The equivalent CLI command is:
+
+```bash
+# Create a forensic copy of the volume 'vol1' from one account to another 
+libcloudforensics aws 'us-east-2b' copydisk --volume_id='vol1' --src_profile='src_profile' --dst_profile='dst_profile' 
+```
+
+If you want to make the copy in a different zone, just specify `dst_zone` in the above method.
+You can also pass a dictionary of tags to add to the volume copy.
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the volume 'vol1' from one account to another, in a different zone
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  dst_zone='us-west-2a',
+                                  volume_id='vol1', 
+                                  src_profile='src_profile', 
+                                  dst_profile='analysis_profile',
+                                  tags={'customTag': 'customValue'})
+```
+
+The equivalent CLI command is:
+
+```bash
+# Create a forensic copy of the volume 'vol1' from one account to another, in a different zone
+libcloudforensics aws 'us-east-2b' copydisk --dst_zone='us-west-2a' --volume_id='vol1' --src_profile='src_profile' --dst_profile='dst_profile' --tags="{'customTag': 'customValue'}" 
+```
+
+Finally, you can opt to pass an `instance_id` parameter to the method instead of a `volume_id`.
+If you do so, then the boot volume of the instance pointed to by `instance_id` will be copied.
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the boot volume of instance 'inst1' in the default account.
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  instance_id='inst1')
+```
+
+The equivalent CLI command is:
+
+```bash
+# Create a forensic copy of the boot volume of instance 'inst1' in the default account
+libcloudforensics aws 'us-east-2b' copydisk --instance_id='inst1'
+```    
 
 #### EBS encryption
 
-TODO
+Making copies of encrypted resources is a bit more involved. By default, encrypted Elastic Block Store (EBS) resources 
+use the aws/ebs key that is associated with the AWS account. Snapshots that are encrypted with this key cannot be shared 
```

Comment:
AWS/EBS if these are acronyms or `aws/ebs` if it's the technical name of the resource

Reply:
It's the technical name (alias of the default AWS managed key)